pkg/config/fetcher/from_processes.go

```diff
@@ -27,3 +27,23 @@ func FetchSecurityAgentConfig(config config.Reader) (string, error) {
 	client := settingshttp.NewClient(c, apiConfigURL, "security-agent")
 	return client.FullConfig()
 }
+
+// TraceAgentConfig fetch the configuration from the process-agent process by querying its HTTPS API
```

Comment:
```suggestion
// TraceAgentConfig fetch the configuration from the trace-agent process by querying its HTTPS API
```

Reply:
It also looks like we're query a http endpoint, not https

Reply:
The debug endpoint of the trace agent doesn't support HTTPS

Reply:
Sorry my suggestion wasn't complete.

```suggestion
// TraceAgentConfig fetch the configuration from the trace-agent process by querying its HTTP API
```